statics/js/components/old_topology_approach.js

```diff
@@ -0,0 +1,3321 @@
+function WSHandler() {
```

Comment:
I don't understand the purpose of merging two files into one big file. It seems to me that you could keep the original files, make modification if needed. It will make the review easier.

Reply:
okay